### PR TITLE
Create Merge Sort Documentation

### LINKED ISSUE #245: My working on documentation
Hi

This is a long time issue to work on some of algorithms documentation.

Thanks

### PR DESCRIPTION
**Pull request by TawfikYasser**
Hi

This PR is my first step in docs.

Thanks

### CODE REVIEW

File: docs/en/Sorting/MergeSort.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+- [C](../../../algorithms/C/sorting/merge-sort.c)
+
+## Video URL
+* [Youtube Video about Merge Sort](https://www.youtube.com/watch?v=jlHkDBEumP0) 
+
+## Others
+[Wikipedia](https://en.wikipedia.org/wiki/Merge_sort)
+
+## Author
+* Java: [@TawfikYasser](https://github.com/TawfikYasser)
```
Comment (ming-tsai) [inline docs/en/Sorting/MergeSort.md:28]: Does this part is necessary?
Remember we have use git for control version

File: docs/en/Sorting/MergeSort.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# Merge Sort
```
Comment (ming-tsai) [inline docs/en/Sorting/MergeSort.md:1]: Could you rename the file's name to `Merge-Sort.md`, after that add it on [README.md](https://github.com/TawfikYasser/DSA/tree/main/docs/en)

**Review by ming-tsai — CHANGES_REQUESTED** [review]
Hi @TawfikYasser,
Nice work, I just add some comments

**Comment by ming-tsai** [comment]
Hi @TawfikYasser,
Are you actvie?

**Review by ming-tsai — APPROVED** [review]
Nice work @TawfikYasser

**Review by ming-tsai — APPROVED** [review]

**Comment by TawfikYasser** [comment]
@ming-tsai  Sorry but I'm very busy these days. Thanks for fixing the issue.

**Pull request merged**